gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -18,7 +18,8 @@
 """
 
 from __future__ import absolute_import
-
+# pylint: disable=unused-import
```

Comment:
nit: please put this in front of the next line.

Reply:
Done.